baseplate/frameworks/pyramid/__init__.py

```diff
@@ -309,39 +296,6 @@ def start_server_span(
         config.add_request_method(start_server_span, "start_server_span")
 
 
-def paste_make_app(_: Dict[str, str], **local_config: str) -> Any:
```

Comment:
:knife: `"paste.app_factory": ["main = baseplate.frameworks.pyramid:paste_make_app"],` in setup.py?

Reply:
done!